person.rb

```diff
@@ -1,3 +1,5 @@
+require_relative('./nameable')
+
 class Person
```

Comment:
- [ ] As demanded by project requirements, kindly make sure the Person class inherits the Nameable class.